WordPress/src/test/java/org/wordpress/android/ui/jpfullplugininstall/onboarding/JetpackFullPluginInstallOnboardingViewModelTest.kt

```diff
@@ -103,6 +105,18 @@ class JetpackFullPluginInstallOnboardingViewModelTest : BaseUnitTest() {
         job.cancel()
     }
 
+    @Test
+    fun `Should track install button click when onInstallFullPluginClick is called`() {
+        classToTest.onInstallFullPluginClick()
+        verify(analyticsTracker).trackInstallButtonClick()
+    }
+
+    @Test
+    fun `Should track dismiss screen click when onDismissScreenClick is called`() {
+        classToTest.onDismissScreenClick()
+        verify(analyticsTracker).trackScreenDismissed()
+    }
+
```

Comment:
Should we add a unit test for the trackScreenShown as well?

Reply:
Thanks for catching that! I missed that one 🤦 (thanks 🙇 )